fe/fe-core/src/main/java/org/apache/doris/insertoverwrite/InsertOverwriteManager.java

```diff
@@ -270,6 +279,50 @@ private boolean rollback(long taskId) {
         return InsertOverwriteUtil.dropPartitions(olapTable, task.getTempPartitionNames());
     }
 
+    /**
+     * If the current table id has a running insert overwrite, throw an exception.
+     * If not, record it in runningTables
+     *
+     * @param dbId Run the dbId for insert overwrite
+     * @param tableId Run the tableId for insert overwrite
+     */
+    public void recordRunningTableOrException(long dbId, long tableId) {
+        runningLock.writeLock().lock();
+        try {
+            if (runningTables.containsKey(dbId) && runningTables.get(dbId).contains(tableId)) {
+                throw new AnalysisException(
+                        String.format("insert overwrite is running on db: %s, table: %s", dbId, tableId));
```

Comment:
```suggestion
                        String.format("Not allowed running Insert Overwrite on same table: %s.%s", dbId, tableId));
                       
```

And I suggest to use db/table name in error msg